src/components/marker-table/index.js

```diff
@@ -122,7 +122,7 @@ class MarkerTree {
 
       displayData = {
         start: _formatStart(marker.start, this._zeroAt),
-        duration: _formatDuration(marker.dur),
+        duration: marker.incomplete ? 'n/a' : _formatDuration(marker.dur),
```

Comment:
Instead of showing "∞ms", make it clear that we don't know the duration. This is more consistent with the marker tooltips which show "unknown duration". I picked 'n/a' since I thought it looked better inside a table.

Reply:
Sounds reasonable. Changed :)